### PR TITLE
Release notes for v1.1.0

### PR DESCRIPTION
**Pull request by ru-fu**
[nRF Connect SDK v1.1.0 Release Notes.pdf](https://github.com/NordicPlayground/fw-nrfconnect-nrf/files/3852295/nRF.Connect.SDK.v1.1.0.Release.Notes.pdf)

### CODE REVIEW

File: doc/release-notes-1.1.0.rst [hunk 1]
```diff
@@ -0,0 +1,621 @@
...
+|NCS| v1.1.0 Release Notes
+##########################
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+|NCS| v1.1.0 supports product development with the nRF9160 Cellular IoT device.
+It contains reference applications, sample source code, and libraries for Bluetooth Low Energy devices in the nRF52 Series, though product development on these devices is not currently supported with the |NCS|.
+
+.. important::
```
Comment (ru-fu) [inline doc/release-notes-1.1.0.rst]: Should we have this note?

File: doc/release-notes-1.1.0.rst [hunk 2]
```diff
@@ -0,0 +1,621 @@
...
+Highlights
+**********
+
+
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.1.0**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
```
Comment (ru-fu) [inline doc/release-notes-1.1.0.rst]: Is this sufficient or should we list all versions?

File: doc/release-notes-1.1.0.rst [hunk 3]
```diff
@@ -0,0 +1,621 @@
...
+
+Use the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See `Updating the nRF9160 DK cellular modem`_ for instructions.
+
+
+Supported boards
+****************
+
+* PCA10090 (nRF9160 DK)
+* PCA20035 (Thingy:91)
```
Comment (ru-fu) [inline doc/release-notes-1.1.0.rst]: Should we list Thingy here?

File: doc/release-notes-1.1.0.rst [hunk 4]
```diff
@@ -0,0 +1,621 @@
...
+* PCA10056 (nRF52840 Development Kit)
+* PCA10059 (nRF52840 Dongle)
+* PCA10040 (nRF52 Development Kit)
```
Comment (ru-fu) [inline doc/release-notes-1.1.0.rst]: What about 52833?

File: doc/release-notes-1.1.0.rst [hunk 5]
```diff
@@ -0,0 +1,621 @@
...
+   * - nRF Connect for Desktop
+     - v3.0.0 or later
+     - `nRF Connect for Desktop`_
+   * - dtc (Linux only)
+     - v1.4.6 or later
+     - :ref:`gs_installing_tools_linux`
+   * - GCC
+     - See :ref:`Installing the toolchain<gs_installing_toolchain_win>`
+     - `GNU Arm Embedded Toolchain`_
+
```
Comment (ru-fu) [inline doc/release-notes-1.1.0.rst]: Please check the versions.

File: doc/release-notes-1.1.0.rst [hunk 6]
```diff
@@ -0,0 +1,621 @@
...
+Change log
+**********
+
+The following sections provide detailed lists of changes by component.
+
+
+???
+===
+
+(? where should these go? where are they defines/documented? what do they do?)
```
Comment (ru-fu) [inline doc/release-notes-1.1.0.rst]: Where should these go and how should they be described?

File: doc/release-notes-1.1.0.rst [hunk 7]
```diff
@@ -0,0 +1,621 @@
...
+
+
+* Added the following libraries:
+
+  * Cloud API (``include/net/cloud.h``) - provides a generic cloud API with an implementation for the nRF Cloud.
+  * :ref:`liblwm2m_carrier_readme` (version 0.8.0) - provides support for the Verizon Wireless network support.
+
+* Added the following drivers:
+
+  * nRF9160 GPS - (? where is that? what does it do?)
```
Comment (ru-fu) [inline doc/release-notes-1.1.0.rst]: ?

File: doc/release-notes-1.1.0.rst [hunk 8]
```diff
@@ -0,0 +1,621 @@
...
+  * Added a function to get the current functional mode.
+  * Added a function to get the current network registration status (not registered, roaming, registered home network).
+  * Added a function to get and set the system mode (LTE-M, NB-IoT, and GPS).
+  * Other minor improvements and fixes.
+
+
+Updated drivers
+---------------
+
+* :ref:`at_cmd_readme`: (? is that a driver or a library?)
```
Comment (ru-fu) [inline doc/release-notes-1.1.0.rst]: ?

File: doc/release-notes-1.1.0.rst [hunk 9]
```diff
@@ -0,0 +1,621 @@
...
+
+BSD library
+-----------
+
+* Updated the :ref:`nrfxlib:bsdlib` to version 0.4.3.
+* Updated bsdlib_init() to return the value of :cpp:func:`bsd_init` instead of (only) zero.
+* Added functionality that overrides untranslated errnos set by the BSD library with a magic word (0xBAADBAAD), instead of EINVAL, and prints a log message.
+  If ASSERTs are enabled, the application will assert.
+* Made DFU socket available through the socket offloading mechanism.
+* PDN socket option translation for in the socket offloading. (? what does this mean?)
```
Comment (ru-fu) [inline doc/release-notes-1.1.0.rst]: ?

File: doc/release-notes-1.1.0.rst [hunk 10]
```diff
@@ -0,0 +1,621 @@
...
+
+* Added the following libraries:
+
+  * :ref:`lib_dfu_target` - abstracts the specific implementation of how a DFU procedure is implemented.
+  * :ref:`fprotect_readme` - uses hardware (BPROT, ACL, or SPU) to protect flash areas from being changed.
+    This library is used by the immutable bootloader.
+
+* Added the following drivers:
+
+  * Flash patch driver - writes to UICR to disable flash patch functionality during the first boot of the image. (? where is this driver?)
```
Comment (ru-fu) [inline doc/release-notes-1.1.0.rst]: ?

File: doc/release-notes-1.1.0.rst [hunk 11]
```diff
@@ -0,0 +1,621 @@
...
+
+
+Updated libraries
+-----------------
+
+* Immutable bootloader: (? which library/libraries are we talking about here?)
```
Comment (ru-fu) [inline doc/release-notes-1.1.0.rst]: ?

File: doc/release-notes-1.1.0.rst [hunk 12]
```diff
@@ -0,0 +1,621 @@
...
+
+  * Created a bl_validate_firmware() function that can be used to ensure that a received upgrade will be accepted by the immutable bootloader.
+    This function is available to be called from subsequent boot steps.
+  * Refactored bl_crypto_client into bl_client. (? what does this mean?)
```
Comment (ru-fu) [inline doc/release-notes-1.1.0.rst]: ?

File: doc/release-notes-1.1.0.rst [hunk 13]
```diff
@@ -0,0 +1,621 @@
...
+
+
+nRF9160
+=======
+
+* Deprecation warning: The nrf_inbuilt_key API in the :ref:`nrfxlib:bsdlib` will be removed in a future release.
+  A replacement library that wraps the AT commands for ``AT%CMNG`` will be available in the |NCS|.
+
+
+.. note::
```
Comment (ru-fu) [inline doc/release-notes-1.1.0.rst]: Please check the old issues - here and in the other sections.

File: doc/release-notes-1.1.0.rst (hunk 1 above)
Comment (nvlsianpu) [reply doc/release-notes-1.1.0.rst]: ^^ good to have such a note here. nRF53 is not in the NCS master at the moment (but will come soon with the upmerge)

File: doc/release-notes-1.1.0.rst [hunk 14]
```diff
@@ -0,0 +1,621 @@
...
+* Bluetooth Low Energy peripheral samples are unstable in some conditions (when pairing and bonding are performed, and when disconnections/re-connections happen).
+
+.. note::
+   Old - which of these are still valid?
+
+    * :ref:`peripheral_lbs` does not report the Button 1 state correctly.
+    * The central samples (:ref:`central_uart`, :ref:`bluetooth_central_hids`) do not support any pairing methods with MITM protection.
+    * On some operating systems, the nrf_desktop application is unable to reconnect to a host.
+    * central_uart: A too long 212-byte string cannot be handled when entered to the console to send to peripheral_uart.
+    * central_hids: After flashing a HEX file to the nrf52_pca10040 board, UART connectivity is lost when using the BLE Controller. The board must be reset to get UART output.
```
Comment (grochu) [inline doc/release-notes-1.1.0.rst]: This works for me, so I think we should remove it. @qung can you still see this issue?

File: doc/release-notes-1.1.0.rst [hunk 15]
```diff
@@ -0,0 +1,621 @@
...
+* The :ref:`nrfxlib:ble_controller` 0.3.0-3.prealpha might assert when receiving a packet with an CRC error on LE Coded PHY after performing a DLE procedure where RX Octets is changed to a value above 140.
+* :option:`CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE` must be set to 2048 when selecting :option:`CONFIG_BT_LL_NRFXLIB` on :ref:`central_uart` and :ref:`central_bas`.
+* :option:`CONFIG_NFCT_IRQ_PRIORITY` must be set to 5 or less when selecting :option:`CONFIG_BT_LL_NRFXLIB` on :ref:`peripheral_hids_keyboard`.
+* When selecting :option:`CONFIG_BT_LL_NRFXLIB`:
+  If a directed high duty cycle advertiser times out, the application might have to wait a short time before starting a new connectable advertiser.
+  Otherwise, starting the advertiser will fail.
+* Bluetooth Low Energy peripheral samples are unstable in some conditions (when pairing and bonding are performed, and when disconnections/re-connections happen).
+
+.. note::
+   Old - which of these are still valid?
```
Comment (grochu) [inline doc/release-notes-1.1.0.rst]: One statement should be removed, see below, but needs a confirmation from Verification.

File: doc/release-notes-1.1.0.rst (hunk 4 above)
Comment (shantha-14) [reply doc/release-notes-1.1.0.rst]: No I dont think we should mention this. Release branch doesnot have nrf52833_pca10100 support.

File: doc/release-notes-1.1.0.rst [hunk 16]
```diff
@@ -0,0 +1,621 @@
...
+In addition to the tools mentioned in :ref:`gs_installing`, the following tool versions are required to work with the |NCS|:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Tool
+     - Version
+     - Download link
+   * - SEGGER J-Link
+     - V6.41
```
Comment (shantha-14) [inline doc/release-notes-1.1.0.rst]: 6.50b

File: doc/release-notes-1.1.0.rst [hunk 17]
```diff
@@ -0,0 +1,621 @@
...
+     - `J-Link Software and Documentation Pack`_
+   * - nRF Command Line Tools
+     - v9.8.1
+     - `nRF Command Line Tools`_
+   * - nRF Connect for Desktop
+     - v3.0.0 or later
```
Comment (shantha-14) [inline doc/release-notes-1.1.0.rst]: v3.3.0 or later

File: doc/release-notes-1.1.0.rst [hunk 18]
```diff
@@ -0,0 +1,621 @@
...
+
+
+
+nRF9160
+=======
+
+* Added the following samples:
+
+  * :ref:`lwm2m_carrier` - demonstrates how to use the :ref:`liblwm2m_carrier_readme` library to connect to the operator LwM2M network.
+  * :ref:`lwm2m_client` - demonstrates how to use LwM2M to connect an nRF9160 DK to an LwM2M server such as Leshan Demo Server via LTE.
```
Comment (lemrey) [inline doc/release-notes-1.1.0.rst]: I suggest that we rephrase this slightly to make it clearer that this sample is using Zephyr's upstream LwM2M stack to implement a sample LwM2M client application; you can't connect to the operator network with this, but only run the upstream LwM2M implementation against a demo server.

File: doc/release-notes-1.1.0.rst (hunk 10 above)
Comment (oyvindronningstad) [reply doc/release-notes-1.1.0.rst]: nrf/drivers/flash_patch
nrf/include/drivers/flash_patch.h

File: doc/release-notes-1.1.0.rst (hunk 8 above)
Comment (lemrey) [reply doc/release-notes-1.1.0.rst]: It's under `/drivers` (yes, debatably).

File: doc/release-notes-1.1.0.rst [hunk 19]
```diff
@@ -0,0 +1,621 @@
...
+* :ref:`at_cmd_readme`: (? is that a driver or a library?)
+
+  * Added an option to initialize the driver manually.
+  * Fixed detection of CME / CMS errors.
+
+
+BSD library
+-----------
+
+* Updated the :ref:`nrfxlib:bsdlib` to version 0.4.3.
```
Comment (lemrey) [inline doc/release-notes-1.1.0.rst]: This release uses `bsdlib` version `0.5.0`.

File: doc/release-notes-1.1.0.rst (hunk 11 above)
Comment (oyvindronningstad) [reply doc/release-notes-1.1.0.rst]: nrf/subsys/bootloader

File: doc/release-notes-1.1.0.rst (hunk 9 above)
Comment (lemrey) [reply doc/release-notes-1.1.0.rst]: I think it means the same as the line above, but for a different socket.
"Made PDN socket available through the socket offloading mechanism."

File: doc/release-notes-1.1.0.rst [hunk 20]
```diff
@@ -0,0 +1,621 @@
...
+* Updated bsdlib_init() to return the value of :cpp:func:`bsd_init` instead of (only) zero.
+* Added functionality that overrides untranslated errnos set by the BSD library with a magic word (0xBAADBAAD), instead of EINVAL, and prints a log message.
+  If ASSERTs are enabled, the application will assert.
+* Made DFU socket available through the socket offloading mechanism.
+* PDN socket option translation for in the socket offloading. (? what does this mean?)
+* Added offloading of local sockets.
```
Comment (lemrey) [inline doc/release-notes-1.1.0.rst]: This one too, fyi.

File: doc/release-notes-1.1.0.rst (hunk 12 above)
Comment (oyvindronningstad) [reply doc/release-notes-1.1.0.rst]: This line should be removed. The PR didn't make it into the release.

File: doc/release-notes-1.1.0.rst (hunk 10 above)
Comment (lemrey) [reply doc/release-notes-1.1.0.rst]: @sigvartmh could you comment on this?

File: doc/release-notes-1.1.0.rst [hunk 21]
```diff
@@ -0,0 +1,621 @@
...
+
+* at_host (``lib/at_host``):
+
+  * Updated to use a dedicated workqueue instead of the system workqueue.
+  * Miscellaneous fixes and improvements.
+
+* :ref:`at_cmd_parser_readme`:
+
+  * Refactored the library.
+  * Added :ref:`at_notif_readme` dispatcher.
```
Comment (lemrey) [inline doc/release-notes-1.1.0.rst]: `at_notif` is a new library.

File: doc/release-notes-1.1.0.rst (hunk 10 above)
Comment (sigvartmh) [reply doc/release-notes-1.1.0.rst]: Not sure what you want me to comment. What oyvind said is true it is a driver  with no header. Which is loaded post_kernel init can be disabled through Kconfig.

File: doc/release-notes-1.1.0.rst (hunk 7 above)
Comment (rlubos) [reply doc/release-notes-1.1.0.rst]: It's [this driver](https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/drivers/nrf9160_gps/nrf9160_gps.c) introduced by https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1029.

It configures GPS in the modem and controls GPS data readout from the modem.

File: doc/release-notes-1.1.0.rst (hunk 8 above)
Comment (rlubos) [reply doc/release-notes-1.1.0.rst]: While arguable, we consider it as a driver.

File: doc/release-notes-1.1.0.rst (hunk 9 above)
Comment (rlubos) [reply doc/release-notes-1.1.0.rst]: @lemrey Could you elaborate on that? Could it be treated as the DFU entry just above (PDN socket support in offloading?).

File: doc/release-notes-1.1.0.rst [hunk 22]
```diff
@@ -0,0 +1,621 @@
...
+Supported boards
+****************
+
+* PCA10090 (nRF9160 DK)
+* PCA20035 (Thingy:91)
+* PCA10056 (nRF52840 Development Kit)
+* PCA10059 (nRF52840 Dongle)
+* PCA10040 (nRF52 Development Kit)
+* PCA10028 (nRF51 Development Kit)
+* PCA63519 (Smart Remote 3 DK add-on)
```
Comment (jhn-nordic) [inline doc/release-notes-1.1.0.rst]: PCA63519 (Smart Remote 3 DK add-on) is there any more.

Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: @jhn-nordic Is there or is not there?

File: doc/release-notes-1.1.0.rst (hunk 7 above)
Comment (evenl) [reply doc/release-notes-1.1.0.rst]: @jtguggedal Maybe you should have a look at this.

File: doc/release-notes-1.1.0.rst [hunk 23]
```diff
@@ -0,0 +1,621 @@
...
+  * Added a function to get and set the system mode (LTE-M, NB-IoT, and GPS).
+  * Other minor improvements and fixes.
+
+
+Updated drivers
+---------------
+
+* :ref:`at_cmd_readme`: (? is that a driver or a library?)
+
+  * Added an option to initialize the driver manually.
```
Comment (evenl) [inline doc/release-notes-1.1.0.rst]: Hasn't that option been in the driver for some time?

Comment (rlubos) [reply doc/release-notes-1.1.0.rst]: It was added while v1.1.0 release was ongoing, it didn't make it to the release branch though:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/870

File: doc/release-notes-1.1.0.rst (hunk 7 above)
Comment (jtguggedal) [reply doc/release-notes-1.1.0.rst]: It's  as @rlubos says. The driver configures the modem for GPS operation, and controls the GPS data coming from the modem. Applications must interact with the GPS using the GPS API and not the driver directly, as it has no public header. The driver is used by the asset tracker. Note that the driver is marked experimental here:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/81646300ceb6aa36faabbb5518075d6cec35ce68/drivers/nrf9160_gps/Kconfig#L9

Comment (jtguggedal) [reply doc/release-notes-1.1.0.rst]: Sorry about the late response, @ru-fu , I overlooked this one.

File: doc/release-notes-1.1.0.rst (hunk 9 above)
Comment (lemrey) [reply doc/release-notes-1.1.0.rst]: Yes, it means we have offloaded the socket options for PDN sockets so I suggested to simply rephrase it like the line above.

File: doc/release-notes-1.1.0.rst (hunk 10 above)
Comment (lemrey) [reply doc/release-notes-1.1.0.rst]: ah, sorry. Øyvind's comment didn't appear when I was reviewing :)

File: doc/release-notes-1.1.0.rst (hunk 23 above)
Comment (lemrey) [reply doc/release-notes-1.1.0.rst]: You mean to say `1.0.0` @rlubos ? :)

Comment (rlubos) [reply doc/release-notes-1.1.0.rst]: @lemrey Correct, thanks for spotting

File: doc/release-notes-1.1.0.rst [hunk 24]
```diff
@@ -0,0 +1,621 @@
...
+* :option:`CONFIG_NFCT_IRQ_PRIORITY` must be set to 5 or less when selecting :option:`CONFIG_BT_LL_NRFXLIB` on :ref:`peripheral_hids_keyboard`.
+* When selecting :option:`CONFIG_BT_LL_NRFXLIB`:
+  If a directed high duty cycle advertiser times out, the application might have to wait a short time before starting a new connectable advertiser.
+  Otherwise, starting the advertiser will fail.
+* Bluetooth Low Energy peripheral samples are unstable in some conditions (when pairing and bonding are performed, and when disconnections/re-connections happen).
+
+.. note::
+   Old - which of these are still valid?
+
+    * :ref:`peripheral_lbs` does not report the Button 1 state correctly.
```
Comment (grochu) [inline doc/release-notes-1.1.0.rst]: This should be removed. @qung any objection?

File: doc/release-notes-1.1.0.rst (hunk 3 above)
Comment (bjda) [reply doc/release-notes-1.1.0.rst]: Yes, it is supported

File: doc/release-notes-1.1.0.rst (hunk 7 above)
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: Updated to:
  * nRF9160 GPS (``drivers/nrf9160_gps/``) - configures the modem for GPS operation and controls the GPS data coming from the modem.
    Applications must interact with the GPS using the GPS API (``includ/gps.h``) and not use the driver directly.

@bjorn-tore-taraldsen @shantha-14 Should we mark it as experimental?

File: doc/release-notes-1.1.0.rst (hunk 8 above)
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: Ok, thx. The docs have it under libraries - I'll make a task to change this.

File: doc/release-notes-1.1.0.rst (hunk 9 above)
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: Updated to:
* Made DFU socket and PDN socket available through the socket offloading mechanism.

File: doc/release-notes-1.1.0.rst (hunk 24 above)
Comment (qung) [reply doc/release-notes-1.1.0.rst]: Yes, it should be removed

File: doc/release-notes-1.1.0.rst (hunk 11 above)
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: Are we mixing updates to the library and updates to the sample here?

File: doc/release-notes-1.1.0.rst (hunk 14 above)
Comment (qung) [reply doc/release-notes-1.1.0.rst]: Yes, we should remove these lines

File: doc/release-notes-1.1.0.rst [hunk 25]
```diff
@@ -0,0 +1,621 @@
...
+
+* :option:`CONFIG_BT_HCI_TX_STACK_SIZE` must be set to 1536 when selecting :option:`CONFIG_BT_LL_NRFXLIB`.
+* The :ref:`nrfxlib:ble_controller` 0.3.0-3.prealpha might assert when receiving a packet with an CRC error on LE Coded PHY after performing a DLE procedure where RX Octets is changed to a value above 140.
+* :option:`CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE` must be set to 2048 when selecting :option:`CONFIG_BT_LL_NRFXLIB` on :ref:`central_uart` and :ref:`central_bas`.
+* :option:`CONFIG_NFCT_IRQ_PRIORITY` must be set to 5 or less when selecting :option:`CONFIG_BT_LL_NRFXLIB` on :ref:`peripheral_hids_keyboard`.
+* When selecting :option:`CONFIG_BT_LL_NRFXLIB`:
+  If a directed high duty cycle advertiser times out, the application might have to wait a short time before starting a new connectable advertiser.
+  Otherwise, starting the advertiser will fail.
+* Bluetooth Low Energy peripheral samples are unstable in some conditions (when pairing and bonding are performed, and when disconnections/re-connections happen).
+
```
[the remark this answers is not in the document]
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: Added as follows:
* When running the :ref:`bluetooth_central_dfu_smp` sample, the :option:`CONFIG_BT_SMP` configuration must be aligned between this sample and the Zephyr counterpart (:ref:`smp_svr_sample`). However, security is not enabled by default in the Zephyr sample.

File: doc/release-notes-1.1.0.rst (hunk 15 above)
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: So the others are all still valid?

File: doc/release-notes-1.1.0.rst (hunk 25 above)
[the remark this answers is not in the document]
Comment (grochu) [reply doc/release-notes-1.1.0.rst]: Looks good

File: doc/release-notes-1.1.0.rst (hunk 15 above)
Comment (grochu) [reply doc/release-notes-1.1.0.rst]: Yes. In the end 2 statements need removing:
- `central_hids: After flashing a HEX file to the nrf52_pca10040 board (...)`, and
-  `peripheral_lbs does not report the Button 1 state correctly.`

File: doc/release-notes-1.1.0.rst (hunk 18 above)
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: Updated to:
  * :ref:`lwm2m_client` - demonstrates how to use Zephyr's :ref:`lwm2m_interface` to implement a sample LwM2M application.
     This sample can run against an LwM2M demo server, but cannot connect to the operator network.

File: doc/release-notes-1.1.0.rst (hunk 20 above)
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: @lemrey So this is covered by "Made DFU socket and PDN socket available ...", or should it be "Made DFU socket, PDN socket, and local sockets available ..."?

File: doc/release-notes-1.1.0.rst (hunk 21 above)
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: Moved up to the section for added libraries:
  * :ref:`at_notif_readme` - dispatches AT command notifications to registered modules.

File: doc/release-notes-1.1.0.rst (hunk 22 above)
Comment (jhn-nordic) [reply doc/release-notes-1.1.0.rst]: not there :)

Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: removed

**Comment by ru-fu** [comment]
OK,  updated according to all comments up to now:
[nRF Connect SDK v1.1.0 Release Notes.pdf](https://github.com/NordicPlayground/fw-nrfconnect-nrf/files/3858952/nRF.Connect.SDK.v1.1.0.Release.Notes.pdf)

Still missing input though:
- Highlights
- Where should the stuff listed in the ``???`` section go?
- Which of the old known issues are still valid? (Props to @grochu for checking the Bluetooth issues ;) )

File: doc/release-notes-1.1.0.rst [hunk 26]
```diff
@@ -0,0 +1,618 @@
...
+Bootloader
+----------
+
+* Public keys are not revoked when subsequent keys are used.
+* The bootloader does not work properly on nRF51.
+
+.. note::
+   Old - which of these are still valid?
+
+    * Building and programming the immutable bootloader (see :ref:`ug_bootloader`) is not supported in SEGGER Embedded Studio.
```
Comment (oyvindronningstad) [inline doc/release-notes-1.1.0.rst]: @hakonfam @andvib Please advise on this and the next.

File: doc/release-notes-1.1.0.rst (hunk 7 above)
Comment (bjorn-tore-taraldsen) [reply doc/release-notes-1.1.0.rst]: No I don't think so. (We do not that for other modules)

File: doc/release-notes-1.1.0.rst [hunk 27]
```diff
@@ -0,0 +1,621 @@
...
+
+
+nRF9160
+=======
+
+* Added the following samples:
+
+  * :ref:`lwm2m_carrier` - demonstrates how to use the :ref:`liblwm2m_carrier_readme` library to connect to the operator LwM2M network.
+  * :ref:`lwm2m_client` - demonstrates how to use LwM2M to connect an nRF9160 DK to an LwM2M server such as Leshan Demo Server via LTE.
+  * :ref:`usb_uart_bridge_sample` - acts as a serial adapter for Thingy:91, exposing 2 UART pairs to a USB host as 2 CDC ACM devices.
```
[the remark this answers is not in the document]
Comment (joakimtoe) [reply doc/release-notes-1.1.0.rst]: CC @jtguggedal @bjda : I agree, we should state that this example is for the nRF52840 on the Thingy:91 HW to provide USB serial ports for debug output and ability to send AT commands to the modem.

File: doc/release-notes-1.1.0.rst (hunk 7 above)
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: ok

File: doc/release-notes-1.1.0.rst [hunk 28]
```diff
@@ -0,0 +1,618 @@
...
+|NCS| v1.1.0 supports product development with the nRF9160 Cellular IoT device.
+It contains reference applications, sample source code, and libraries for Bluetooth Low Energy devices in the nRF52 Series, though product development on these devices is not currently supported with the |NCS|.
+
+.. important::
+   |NCS| v1.1.0 does not provide support for nRF5340.
+   For nRF5340 support, check out the master branch.
+
+Highlights
+**********
+
```
Comment (ru-fu) [inline doc/release-notes-1.1.0.rst]: Need content here.

File: doc/release-notes-1.1.0.rst [hunk 29]
```diff
@@ -0,0 +1,621 @@
...
+.. note::
+   Old - which of these are still valid?
+
+   * In the BSD library, the GNSS sockets implementation is experimental.
+
+   * Forcing a cold start and writing AGPS data is not yet supported.
+
+nrfx 1.7.1
+==========
+
```
[the remark this answers is not in the document]
Comment (barsok) [reply doc/release-notes-1.1.0.rst]: @bjorn-tore-taraldsen 2 first issues of the 3 are valid. The 3rd one is corrected now.

File: doc/release-notes-1.1.0.rst [hunk 30]
```diff
@@ -0,0 +1,621 @@
...
+(new) PAW3212 driver
+(new) nrf52810_pca20045 board
+(new) nrf52_pca20044 board
+
+adp536x driver:
+
+    Added buck discharge resistor configuration.
+
+
```
[the remark this answers is not in the document]
Comment (barsok) [reply doc/release-notes-1.1.0.rst]: Change log is there:
https://github.com/NordicSemiconductor/nrfx/blob/v1.8.1/CHANGELOG.md
Is this ok?

File: doc/release-notes-1.1.0.rst (hunk 27 above)
[the remark this answers is not in the document]
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: Updated to:
  * :ref:`usb_uart_bridge_sample` - acts as a serial adapter for Thingy:91, providing USB serial ports for debug output and the ability to send AT commands to the modem.
    This sample runs on the nRF52840 SoC on Thingy:91.

File: doc/release-notes-1.1.0.rst (hunk 30 above)
[the remark this answers is not in the document]
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: Added a section - please check.

File: doc/release-notes-1.1.0.rst (hunk 26 above)
Comment (andvib) [reply doc/release-notes-1.1.0.rst]: I did some quick testing just now, and it seems they're both kinda valid.

From before I had only tested with SES in Windows for nrf9160_pca10090ns, which works. For nrf9160_pca10090 and nrf52840_pca10056 however it did not work. The immutable bootloader works for all the three boards when I compiled it with GCC in Linux.

So to me it seems like the safest and clearest will be to keep both of these points.

File: doc/release-notes-1.1.0.rst (hunk 28 above)
Comment (hakonfam) [reply doc/release-notes-1.1.0.rst]: I would highlight support for modem FOTA.

Comment (frkv) [reply doc/release-notes-1.1.0.rst]: New zephyr driver for CC310 HW initialization and entropy
nrf_security: Passing NIST CAVP tests for nRF9160 and nRF52840 using CryptoCell CC310 + SW
nrf_security: Passing handshake of all major suites on TLS and DTLS using CryptoCell CC310 + SW

**Comment by ru-fu** [comment]
[nRF Connect SDK v1.1.0 Release Notes.pdf](https://github.com/NordicPlayground/fw-nrfconnect-nrf/files/3862884/nRF.Connect.SDK.v1.1.0.Release.Notes.pdf)

**Comment by ru-fu** [comment]
[nRF Connect SDK v1.1.0 Release Notes.pdf](https://github.com/NordicPlayground/fw-nrfconnect-nrf/files/3863784/nRF.Connect.SDK.v1.1.0.Release.Notes.pdf)

File: doc/release-notes-1.1.0.rst (hunk 19 above)
[the remark this answers is not in the document]
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: discussed, will keep the versions

File: doc/release-notes-1.1.0.rst [hunk 31]
```diff
@@ -0,0 +1,587 @@
...
+     - `nRF Connect for Desktop`_
+   * - dtc (Linux only)
+     - v1.4.6 or later
+     - :ref:`gs_installing_tools_linux`
+   * - GCC
+     - See :ref:`Installing the toolchain<gs_installing_toolchain_win>`
+     - `GNU Arm Embedded Toolchain`_
+
+
+As IDE, we recommend to use |SES| (Nordic Edition), version 4.20a or later.
```
Comment (shantha-14) [inline doc/release-notes-1.1.0.rst]: we should remove "or later" we have tested only with 4.20a

File: doc/release-notes-1.1.0.rst [hunk 32]
```diff
@@ -0,0 +1,621 @@
...
+   :header-rows: 1
+
+   * - Tool
+     - Version
+     - Download link
+   * - SEGGER J-Link
+     - V6.41
+     - `J-Link Software and Documentation Pack`_
+   * - nRF Command Line Tools
+     - v9.8.1
```
[the remark this answers is not in the document]
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: updated

File: doc/release-notes-1.1.0.rst (hunk 31 above)
Comment (ru-fu) [reply doc/release-notes-1.1.0.rst]: Removed. But it probably won't be available from the provided links anymore once we get an updated SES.

**Pull request merged**